src/app/map/map-page.component.scss

```diff
@@ -186,6 +186,11 @@
     left: 50%;
     transform: translate(-50%, 0);
   }
+
+  .map-container__map {
+    z-index: z-index-variables.$map-container;
```

Comment:
What's the reason for having the map with a z-index of -1?

I think you could a comment, maybe in the z-index-variables File.

Reply:
I needed to make sure that the map came last in the tab order. Setting it to 0 was not enough. I will add a comment